stdlib/functools.pyi

```diff
@@ -11,6 +11,10 @@ _AnyCallable = Callable[..., Any]
 
 _T = TypeVar("_T")
 _S = TypeVar("_S")
+_P1 = ParamSpec("_P1")
+_R1 = TypeVar("_R1")
+_P2 = ParamSpec("_P2")
+_R2 = TypeVar("_R2")
```

Comment:
I think it could be confusing to rename these `_WrapperP` and `_WrappedP`. `_P1` is used as a type argument in both the `_Wrapped` and `_Wrapper` types in the `wraps` signature. It's used to translate between the wrapped and the wrapper.

You mentioned that you ran into inconsistencies. Can you elaborate?